### PR TITLE
Allow user to specify AZURE_CLOUD_PROVIDER_ROOT when running ci-entrypoint locally

### PR DESCRIPTION
**Pull request by marosset**
…

 

 
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
Let  user specify AZURE_CLOUD_PROVIDER_ROOT env var when running ci-entrypoint locally

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**TODOs**:

- [ ] squashed commits
- [ ] includes documentation
- [ ] adds unit tests

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by CecileRobertMichon** [comment]
Commit / PR title has a typo, otherwise lgtm

**Comment by marosset** [comment]
> Commit / PR title has a typo, otherwise lgtm

Fixes both :-/

File: scripts/ci-build-azure-ccm.sh [hunk 1]
```diff
@@ -38,12 +38,18 @@ export CNM_IMAGE_NAME=azure-cloud-node-manager
 export WINDOWS_IMAGE_VERSION=1809
 declare -a IMAGES=("${CCM_IMAGE_NAME}" "${CNM_IMAGE_NAME}")
 
+
```
Comment (CecileRobertMichon) [inline scripts/ci-build-azure-ccm.sh:41]: nit: extra whiteline

**Review by CecileRobertMichon — COMMENTED** [review]
lgtm

**Comment by CecileRobertMichon** [comment]
Failing repetitively on #3227 :(

/retest

**Pull request merged**